src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -105,6 +112,8 @@ protected Colony(int id, int dim)
     {
         this.id = id;
         this.dimensionId = dim;
+
+        this.colonyAchievements = new ArrayList<Achievement>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2293)